OmiseSDK/CardExpiryDateTextField.swift

```diff
@@ -205,7 +205,7 @@ public class CardExpiryDateTextField: OmiseTextField {
             return
         }
         
-        let currentAttributes = defaultTextAttributes
+        let _currentAttributes = defaultTextAttributes
```

Comment:
We didn't use this property. We can remove this line.

Reply:
Removed